### DISCUSSION TITLE
Cold shoe support

### DISCUSSION THREAD

**Discussion by averydelmiller** [edgarkech/scalable_field_camera#13] [Q&A]
Hello! I'm just in love with the idea of this camera, I can't wait to give it my own go.
I was thinking about cold shoe mounts, and I noticed in some of the attached images there are two empty holes-- are these for something I'm missing, or are they some kind a optional mounting points?
Thanks so much!

**Comment by Egorarchitec** [comment]
Two empty holes are for stopping the lid to open (you close the lid, turn the knob a bit to extend the rail so it goes into the holes and lock it. The cold shoe is very easy to attach in any 3D program especially when you have the model of cold shoe. Either as an integrated cold shoe, or attachable (needs to add screw holes into the body).

**Comment by THY3118** [comment]
I found that there is no model of belowsframes in the folder of 4X5. Can you tell me where it is or can you add it?

**Reply by edgarkech** [reply]
Quoting from the README.md file in the STL_4x5 folder (also available here: https://github.com/edgarkech/scalable_field_camera/blob/master/STL_4x5/README.md):

> bellows
History has shown that standardcameras is sometimes out of stock for the bellows.

>Alternative 1: build your own bellows (sorry, no howto from me, but there are lots of excellent sources!)
>Alternative 2: Find a replacement bellows for another field camera (e.g. Linhof Technika 4x5, Toyo 45A, Horseman, HD/HF/FA 45, ...).
>Make sure to get squared bellows with a front measurement of something between 88 and 98mm (default is 94mm) and a back measurement of something between 141 and 153mm (default is 144mm).

>You have to create your own STLs for the bellow frames from the according SCAD files. In the config_4x5.scad file find the variables "vBellowsFrontInner_l" and "vBellowsBackInner_l" and change them to the measurements of your bellows - don't change anything else unless you have a really good reason for it. Let OpenSCAD render your SCAD files and export to STL.

>bellows frames (in case you are not using bellows from standard cameras):
>The bellows frames consist of two individual parts (inner and outer frame), but I modeled them as one piece for easier modeling and printing. They are just "linked" together via the bottom 0.1mm (for easier printing). Just use a knife to split inner and outer frame and clean up the edges.
